src/documents/ModelRelay.md

```diff
@@ -134,20 +145,48 @@ This is done using the `modelRelay` section of a component's top-level options.
         </tr>
```

Comment:
Since I cannot add comment on line 141 which isn't modified in this pull request, leave the comment for it here on this closest line. 

I kind of remember you mentioned `modelRelay` block does not support nested transforms, which my understanding was that `transform` option is not supported. Am I wrong?